docs/endpointslice-metrics.md

```diff
@@ -6,5 +6,6 @@
 | kube_endpointslice_info        | Gauge       |                                                                                                                           | `endpointslice`=&lt;endpointslice-name&gt; <br> `namespace`=&lt;endpointslice-namespace&gt;                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | EXPERIMENTAL |
 | kube_endpointslice_ports       | Gauge       |                                                                                                                           | `endpointslice`=&lt;endpointslice-name&gt; <br> `namespace`=&lt;endpointslice-namespace&gt; <br> `port_name`=&lt;endpointslice-port-name&gt; <br> `port_protocol`=&lt;endpointslice-port-protocol&gt; <br> `port_number`=&lt;endpointslice-port-number&gt;                                                                                                                                                                                                                                                                                                                      | EXPERIMENTAL |
 | kube_endpointslice_endpoints   | Gauge       |                                                                                                                           | `endpointslice`=&lt;endpointslice-name&gt; <br> `namespace`=&lt;endpointslice-namespace&gt; <br> `ready`=&lt;endpointslice-ready&gt; <br> `serving`=&lt;endpointslice-serving&gt; <br> `terminating`=&lt;endpointslice-terminating&gt; <br> `hostname`=&lt;endpointslice-hostname&gt; <br> `targetref_kind`=&lt;endpointslice-targetref-kind&gt; <br> `targetref_name`=&lt;endpointslice-targetref-name&gt; <br> `targetref_namespace`=&lt;endpointslice-targetref-namespace&gt; <br> `nodename`=&lt;endpointslice-nodename&gt; <br> `endpoint_zone`=&lt;endpointslice-zone&gt; | EXPERIMENTAL |
+| kube_endpointslice_endpoints_hints   | Gauge       |  Each line is a hint applied to an endpoint-slice                                                                   | `endpointslice`=&lt;endpointslice-name&gt; <br> `namespace`=&lt;endpointslice-namespace&gt; <br> `address`=&lt;endpointslice-address[0]&gt;  <br> `hostname`=&lt;endpointslice-hostname&gt; <br> `endpoint_zone`=&lt;endpointslice-zone&gt; <br> `hint`=&lt;endpointslice-hint&gt; | EXPERIMENTAL |
```

Comment:
Small nit on the metric name:
```suggestion
| kube_endpointslice_hints   | Gauge       |  Each line is a hint applied to an endpointslice                                                                   | `endpointslice`=&lt;endpointslice-name&gt; <br> `namespace`=&lt;endpointslice-namespace&gt; <br> `address`=&lt;endpointslice-address[0]&gt;  <br> `hostname`=&lt;endpointslice-hostname&gt; <br> `endpoint_zone`=&lt;endpointslice-zone&gt; <br> `hint`=&lt;endpointslice-hint&gt; | EXPERIMENTAL |
```

Reply:
I don't think we should change the metric name. It is not the hints of the endpointslice but the hints of the endpoints that are part of the endpointslice.

Reply:
Fair point. :+1: